src/e3/anod/spec.py

```diff
@@ -274,6 +274,25 @@ def enable_name_generator(self) -> bool:
         """
         return False
 
+    @property
+    def readme_filename(self) -> str:
+        """The readme filename to use.
+
+        :return: The readme filename to use.
+            Note that the return value cannot be None or an empty string
```

Comment:
You may want to use the ``.. note::`` RST markup:

```
:return: The readme filename to use.

.. note:: The return value cannot be None or an empty string.
```

Reply:
Done.